iaso/models/base.py

```diff
@@ -729,17 +729,11 @@ def for_filters(
             queryset = queryset.filter(org_unit_id=org_unit_id)
 
         if org_unit_parent_id:
```

Comment:
Is the use of this parameter actually tested? If not, that would deserve to be done sooner than later.